doc/source/monitoring.rst

```diff
@@ -32,12 +164,166 @@ into ElasticSearch/InfluxDB from where it then can be visualised using Kibana/Gr
+
+    pip install --upgrade pip
+    pip install elasticsearch 
+    wget https://files.pythonhosted.org/packages/52/7e/22ca617f61e0d5904e06c1ebd5d453adf30099526c0b64dca8d74fff0cad/stomp.py-4.1.22.tar.gz
+    tar -zxvf stomp.py-4.1.22.tar.gz
+    cd stomp.py-4.1.22
+    python setup.py install
+
+Change the configurations (message broker and elastisearch cluster) in exporter.py and start it. It could be made as a systemd service for convenience.
+
+5) Create Kibana dashboards based on the imported messages.
+
+A set of pre-defined dashboards can be found `here <https://github.com/feipengsy/rucio-monitoring/tree/master/kibana/visualization>`__. in Jason format which could be imported to Kibana directly. But you may have to resolve different UUIDs in Kibana.
+
+-----------------
 Access monitoring
 -----------------
 
-The traces are sent by the pilots or the rucio clients whenever a file is downloaded/uploaded.
+The traces are sent by the pilots or the rucio clients whenever a file is downloaded/uploaded. This is simillar with the data transferring monitoring.
+
+----------------------
+Rucio database dumping
+----------------------
+
+Besides the internal, data transferring/deletion/accessing monitoring, it's also possible to dump the Rucio internal database directly to Elasticsearch. Then information like data location, accounting, RSE summary could be visualised using Kibana or Grafana. 
+
+We provide several examples of dumping Rucio DB tables using the logstash jdbc plugin and making plots based on them.
+
+To start a logstash pipeline, run::
+ 
+    logstash -f rse.conf
+
+Where the rse.conf contains::
+ 
+    input {
+      jdbc {
+        jdbc_connection_string => ""
+        jdbc_user => ""
+        jdbc_password => ""
+        jdbc_driver_library => "/usr/share/logstash/java/postgresql-42.2.6.jar"
+        jdbc_driver_class => "org.postgresql.Driver"
+        statement => "SELECT rses.rse, rse_usage.source, rse_usage.used, rse_usage.free, rse_usage.files FROM rse_usage INNER JOIN rses ON rse_usage.rse_id=rses.id WHERE rse_usage.files IS NOT NULL AND rse_usage.files!=0;"
+      }
+    }
+    output {
+      elasticsearch {
+        hosts => [""]
+        action => "index"
+        index => "rucio_rse"
+        user => ""
+        password => ""
+      }
+    }
+
+The rse pipeline dumps data like how large is the total space, how large is the used space, how many files are saved on each RSE etc. Please fill in the jdbc connection details and Elastisearch connection details in the config file.
+
+More pipeline definitions can be found `here <https://github.com/feipengsy/rucio-monitoring/tree/master/scripts/logstash>`__, and users could design their own DB queries for their specific monitoring needs. Also users could directly import the Elasticsearch indices and Kibana dashboard from `these <https://github.com/feipengsy/rucio-monitoring/tree/master/kibana/visualization/db_dump>`__ jason files.
```

Comment:
Please change this a bit in a way to point to your `rucio-monitoring` toolkit as an example. I think it is really a great tool and should definitely be mentioned here, I just don't want to create the impression that this is the only way to do it - I don't think you are saying that in the text, I would just still try to make it a bit clearer.